src/runtime/substitution.cc

```diff
@@ -1197,8 +1193,8 @@ void Graph::export_strategy_computation_graph(
   GraphStructure<Graph> s;
 
   for (auto const &node : s.get_nodes(*this)) {
+    // Add node
     if (strategy.find(node) == strategy.end()) {
-      dot.add_node(node, {{"label", node.to_string()}});
```

Comment:
Why delete this?

Reply:
Oh, I think is is in the else branch below. This should be correct.